docs/repo/property-pages/property-specification.md

```diff
@@ -195,6 +195,24 @@ We use this on the `LangVersion` property, for example, as this value is intenti
 </StringProperty>
 ```
 
+### Name/Value List
+
+When a property contains a variable number of name/value pairs, you can use the `NameValueList` editor on `StringProperty` to display a two-column grid in the UI that allows users to edit values and add/remove rows.
+
+The property's string value should be encoded with format resembling `A=1,B=2`, using `/` as an escape character if needed. See `LaunchProfileEnvironmentVariableEncoding` in this repo for further details.
```

Comment:
Forward slash (not backslash) is the escape character?

Reply:
That's right, in this case.